datalad_dataverse/add_sibling_dataverse.py

```diff
@@ -43,15 +43,18 @@ class AddSiblingDataverse(ValidatedInterface):
     In order to be able to use this command, a personal access token has to be
     generated on the Dataverse platform. You can find it by clicking on your
     name at the top right corner, and then clicking on Api Token>Create Token.
+
+    Furthermore, a dataset on such a dataverse instance has to already exist in
+    order to add it as a sibling to a DataLad dataset.
     """
 
     _examples_ = [
-        dict(text="Create a dataverse dataset sibling for sharing and citing",
+        dict(text="add a dataverse dataset sibling for sharing and citing",
              code_py="""\
                  > ds = Dataset('.')
-                 > ds.add_sibling_dataverse(url='https://demo.dataverse.org', name='dataverse')
+                 > ds.add_sibling_dataverse(url='https://demo.dataverse.org', name='dataverse', ds_pid='https://doi.org/10.5072/FK2/PMPMZM')
```

Comment:
A quick test confirms that the doi can't have an https prefix:

```
 datalad add-sibling-dataverse -s smolly https://demo.dataverse.org https://doi:10.70122/FK2/NQPP6A --mode annex --credential demo-dataverse
CommandError: 'git -c diff.ignoreSubmodules=none annex initremote smolly-storage type=external externaltype=dataverse url=https://demo.dataverse.org doi=https://doi:10.70122/FK2/NQPP6A exporttree=no encryption=none credential=demo-dataverse -c annex.dotfiles=true' failed with exitcode 1 under /tmp/myds
initremote smolly-storage 
failed
git-annex: external special remote error: ERROR: GET HTTP 404 - https://demo.dataverse.org/api/v1/datasets/:persistentId/?persistentId=doi:https://doi:10.70122/FK2/NQPP6A. MSG: {"status":"ERROR","message":"Dataset with Persistent ID doi:https://doi:10.70122/FK2/NQPP6A not found."}
initremote: 1 failed
```


```
datalad add-sibling-dataverse -s smolly https://demo.dataverse.org doi:10.70122/FK2/NQPP6A --mode annex --credential demo-dataverse 
add_sibling_dataverse.storage(ok): . [smolly-storage: https://demo.dataverse.org (DOI: doi:10.70122/FK2/NQPP6A)]
[INFO   ] Configure additional publication dependency on "smolly-storage" 
add_sibling_dataverse(ok): . [smolly: datalad-annex::?type=external&externaltype=dataverse&encryption=none&exporttree=no&url=https%3A//demo.dataverse.org&doi=doi:10.70122/FK2/NQPP6A&credential=demo-dataverse (DOI: doi:10.70122/FK2/NQPP6A)]

```
```

Reply:
Nevermind, I malformatted the URL :) 
Maybe an example with the other DOI format doesn't hurt, though, that would be what dataverse metadata would provide copy-pastable

Reply:
Will do. URL is copy-pastable, too, though - it's right at the top of the dataset's page